src/store.c

```diff
@@ -263,21 +302,21 @@ static bool writer_open(
 
     store->vma_len = to_vma_len(len);
     store->vma = mmap(NULL, store->vma_len, PROT_WRITE | PROT_READ, MAP_SHARED, store->fd, 0);
+
```

Comment:
why the extra line? Are you saying that my code formatting is somewhat flawed? I'm insulted!

Also, brr brrr...

Reply:
I removed the line